ndohyep/templates/core/article_page.html

```diff
@@ -1,4 +1,4 @@
-{% extends "base.html" %}
+{% extends "core/_article_page.html" %}
```

Comment:
Can you explain the reasoning behind subclassing this extra template instead of `base.html` ?